main.py

```
@@ -123,10 +125,101 @@ def __init__(self, context: star.Context, config: AstrBotConfig) -> None:
+    def _start_web_admin_server(self) -> None:
+        web_config = self.config.get("web_config", {})
+        if isinstance(web_config, dict) and web_config.get("enabled") is False:
+            logger.info("Telegram Forwarder Web 管理页面未启用。")
+            return
+
+        try:
+            from .core.web_admin import WebAdminServer
+
+            self.web_admin_server = WebAdminServer(self, self._web_loop)
+            self.web_admin_server.start()
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)